integration/tests/karpenter/karpenter_test.go

```diff
@@ -26,34 +28,25 @@ func init() {
 	if err := api.Register(); err != nil {
 		panic(fmt.Errorf("unexpected error registering API scheme: %w", err))
 	}
-	params = tests.NewParams("")
+	params = tests.NewParams("karp")
 }
 
 func TestKarpenter(t *testing.T) {
 	testutils.RegisterAndRun(t)
 }
 
 var _ = Describe("(Integration) Karpenter", func() {
-
-	var (
-		clusterName string
-	)
-
-	BeforeEach(func() {
-		// the randomly generated name we get usually makes one of the resources have a longer than 64 characters name
-		clusterName = fmt.Sprintf("it-karpenter-%d", time.Now().Unix())
-	})
-
 	Context("Creating a cluster with Karpenter", func() {
 		It("should support karpenter", func() {
 			cmd := params.EksctlCreateCmd.
 				WithArgs(
 					"cluster",
 					"--config-file=-",
 					"--verbose=4",
+					"--kubeconfig", params.KubeconfigPath,
```

Comment:
I ran it so many times and this wasn't working. It gave the localhost error thingy. I had to do the write kubeconfig stuff otherwise it failed in CI.

But I hope it will work. :D

Reply:
I'm fairly certain it should be okay!

Reply:
I was too! :DDD